packages/web3-providers-ws/types/index.d.ts

```diff
@@ -20,14 +20,6 @@
  * @date 2018
  */
 
-import {WebsocketProviderBase} from 'web3-core-helpers';
-
-export {
-    WebsocketProviderOptions,
-    ReconnectOptions,
-    JsonRpcResponse,
-    JsonRpcPayload,
-    RequestItem
-} from 'web3-core-helpers';
+import { WebsocketProviderBase } from 'web3-core-helpers';
```

Comment:
The additional exports can still stay because it isn't any logical change and does improve the TS DX.